src/AutoWebhook.php

```diff
@@ -63,6 +64,15 @@ public function autoEnableWebhook($key_id, $key_secret)
                 ['absolute' => TRUE]
             )->toString();
 
+            if($key_id == null || $key_secret == null)
```

Comment:
@garimaprachi use or instead of ||

Reply:
@garimaprachi triggerValidationInstrumentation should be in validateConfigurationForm()